admin-dev/themes/new-theme/scss/components/layout/_content_div.scss

```diff
@@ -38,6 +38,14 @@
   }
 }
 
+// Special case for customer page the table has too many columns to fit the screen so we allow an horizontal scroll
```

Comment:
I think the same issue can happen on any page, providing it has enough columns 🤔 

I tried with "languages", you can see the issue with a specific screen size

Reply:
Yes it can, but this is a quick fix just for the customer page because it can happen on many screens
For other pages we need to think about a more permanent fix for all pages, but it will be done in 177 This is what we decided with @TristanLDD

Reply:
Makes sense 👍